### ISSUE TITLE
Android dir changed on chrome os v 108/109

### ISSUE THREAD

**Issue by moddedstuffguy** [CerisWhite/CrAnberry#27]
Now android dir is /opt/google/vms/android
Please, fix the script as soon as possible if you are able to do so.
Thx

**Comment by moddedstuffguy** [comment]
I am in chromeos with android version 11

**Comment by CerisWhite** [comment]
Sorry about the delay in response. Can you run `ls /opt/google/vms/android/` for me and paste the output here? I'll update the script to try working with it when you do. Can't guarantee it'll actually work though, since I don't have a chromebook to test with at the moment

**Comment by moddedstuffguy** [comment]
Ok, no problem. Now i am not able to do so (busy right now). I'll do that later when I finish doing something.

**Comment by moddedstuffguy** [comment]
Ok, No problem.
Files inside that DIR:
system.raw.img — ureadahead.pack — vendor.raw.img — vmlinux

**Comment by CerisWhite** [comment]
No `rootfs` folder of any kind?

**Comment by moddedstuffguy** [comment]
Oh it could be because i didnt ran the script (as i formatted my chromebook some days ago

**Comment by moddedstuffguy** [comment]
I'll try latef

**Comment by moddedstuffguy** [comment]
Nope, no folders inside

**Comment by moddedstuffguy** [comment]
only the compressed img file, the pack file, the vendor img, and vmlinux

**Comment by CerisWhite** [comment]
Do you have any idea where the `rootfs` directory containing the `android_data` directory might be on newer systems? That location is essential to the operation of the script.

**Comment by moddedstuffguy** [comment]
no, i could investigate a bit. If i find something, I'll tell you

**Comment by moddedstuffguy** [comment]
@CerisWhite
I did not found `rootfs` dir by using
```chronos@localhost
chronos@localhost $/ find rootfs -type dir

**Comment by moddedstuffguy** [comment]
I'll investigate more in a few hours. I'll do an exhaustive search of that dir. I will try as well to use `adb devices` *(to connect to android subsystem's adb)* and `adb shell` *(to see android files)* to see if that can help me to find `rootfs` Android's dir

**Comment by supechicken** [comment]
AFAIK ARCVM no longer mount the rootfs directory to Chrome OS shell since the rootfs is used in Android VM only

**Comment by moddedstuffguy** [comment]
I found the `arcvm` folder inside **/etc/**
There are no relevant contents inside. Only a manifest file called `contents.json`

SELinux dir is inside **/etc/** *(see the attached screenshot)*
![CHRONOS SCREENSHOT](https://user-images.githubusercontent.com/114933191/221792419-7bf88226-b52b-4437-9378-fde80d55c8e3.png)

There is no rootfs inside.
I am still searching for the it. If I find something, I'll tell you @CerisWhite

**Comment by moddedstuffguy** [comment]
I FOUND `android-data` DIRECTORY!
It says for me permission denied when using `cd /var/run/arcvm/android-data`

I am using `adb shell` to investigate inside android subsystem

**Comment by moddedstuffguy** [comment]
I accessed android-data by doing:
```change perms with sudo
sudo chmod 777 /var/run/arcvm/android-data/
```

**Comment by CerisWhite** [comment]
`adb shell` isn't going to help us much. We need to find out where the system is being mounted to. You can probably do this fastest by running `mount | grep loop` and searching for android-related directory names.

**Comment by moddedstuffguy** [comment]
Ok, i mounted `loop`
Now what?

**Comment by CerisWhite** [comment]
I didn't say to mount anything.

Running the mount command by itself is supposed to produce a list of currently mounted devices, including the loop devices we need to look at.

**Comment by moddedstuffguy** [comment]
There is no `loop` device
I've run `mount` to see manually if there is any loop device and there is no one:
I will not attach the output as it contains some of my files' names.

**Comment by eliyoung540** [comment]
> I didn't say to mount anything.
> 
> Running the mount command by itself is supposed to produce a list of currently mounted devices, including the loop devices we need to look at.

i found the android_data directory

adb       app-ephemeral  bootchart     gsi          mediadrm  ota          resource-cache             system      user_de
anr       app-lib        cache         incremental  misc      ota_package  rollback                   system_ce   vendor
apex      app-private    dalvik-cache  local        misc_ce   per_boot     rollback-observer          system_de   vendor_ce
app       app-staging    data          lost+found   misc_de   preloads     server_configurable_flags  tombstones  vendor_de
app-asec  backup         drm           media        nfc       property     ss                         user

**Comment by eliyoung540** [comment]
> 

it can be found at/home/root/210f0256d4eb5863f36bb043a3ad5dee2214cc21/android-data

**Comment by macccorehenni** [comment]
@eliyoung540 that is a hash; I reccomend going into your /home/root folder first
